docs/book/src/tasks/experimental-features/experimental-features.md

```diff
@@ -0,0 +1,68 @@
+# Experimental Features
+
+Cluster API now ships with a new experimental package that lives under exp/ directory which has new features. This is a
+temporary location for features which will be moved to their permanent locations after graduation. Users can experiment with these features by enabling them using feature gates.
```

Comment:
I would add a note about how the features in the exp/ package come with no guarantee, similar to the note with have in the exp/ package readme:

```
Packages here are experimental and unreliable. Some may one day be promoted to the main repository, or they may be modified arbitrarily or even disappear altogether.

In short, code in this subrepository is not subject to any compatibility or deprecation promise.
```